src/test/java/com/puppycrawl/tools/checkstyle/CheckerTest.java

```diff
@@ -1323,7 +1328,7 @@ public void testDefaultLoggerClosesItStreams() throws Exception {
                 new CloseAndFlushTestByteArrayOutputStream()) {
             checker.setModuleClassLoader(Thread.currentThread().getContextClassLoader());
             checker.addListener(new DefaultLogger(testInfoOutputStream,
-                true, testErrorOutputStream, true));
```

Comment:
Should  this be done in separate issue? The ctor for `DefaultLogger` is not affected by this PR.

Reply:
Done.

Splitting up previous 1 PR into many. Missed that this file contained multiple fixes in 1.